core/src/main/scala/org/broadinstitute/dsde/rawls/model/SpendReportingModel.scala

```diff
@@ -52,7 +52,7 @@ object SpendReportingAggregationKeys {
   }
 
   case object Daily extends SpendReportingAggregationKey {
-    override val bigQueryField: String = "DATE(_PARTITIONTIME)"
+    override val bigQueryField: String = "DATE(REPLACE_TIME_PARTITION_COLUMN)"
```

Comment:
I don't love this, but I tried pushing the configuration value down into this class and it was an absolute mess. This approach allows us to keep the logic of "use default config value or not" in a single place-- https://github.com/broadinstitute/rawls/pull/1963/files#diff-6bdd7e72eedead7aafdcd9b71fc64d8eaf45c390b2db2d004f768ea07a43fc53R220